lib/is_it_working/handler.rb

```diff
@@ -107,6 +111,10 @@ def synchronize
       end
     end
 
+    def timer(*args, **kwargs)
```

Comment:
I decided to go all in on keyword arguments, which means no support for ruby < 2.0. I feel fine with that, but curious if that gives anyone pause...